gemhardware/src/common/amc13/AMC13Manager.cc

```diff
@@ -307,22 +307,26 @@ void gem::hw::amc13::AMC13Manager::initializeAction()
   gem::utils::LockGuard<gem::utils::Lock> guardedLock(m_amc13Lock);
 
   // enable daq link (if SFP mask is non-zero
-  DEBUG("Enabling DAQLink with settings: fake data:" << m_enableFakeData
+  DEBUG("AMC13Manager::initializeAction Enabling DAQLink with settings: fake data:" << m_enableFakeData
         << ", sfpMask:" << m_sfpMask);
 
   p_amc13->fakeDataEnable(m_enableFakeData);
   p_amc13->daqLinkEnable(m_enableDAQLink);
 
   // enable SFP outputs based on mask configuration
   p_amc13->sfpOutputEnable(m_sfpMask);
-  /*
+
   // ignore AMC tts state per mask
-  if (m_ignoreAmcTts) {
-    p_amc13->write(amc13::AMC13Simple::T1, "CONF.AMC.TTS_DISABLE_MASK", 0xfff);
+  INFO("AMC13Manager::initializeAction m_amcIgnoreTTSList " << m_amcIgnoreTTSList);
+  m_ignoreAMCTTS = p_amc13->parseInputEnableList(m_amcIgnoreTTSList,true);
+  INFO("AMC13Manager::initializeAction m_amcIgnoreTTSList " << m_amcIgnoreTTSList
+       << " parsed as m_ignoreAMCTTS: " << std::hex << m_ignoreAMCTTS << std::dec);
+  if (m_ignoreAMCTTS) {
```

Comment:
what's the difference between `m_amcIgnoreTTSList` and `m_amcIgnoreTTS`? Do we really need both?

Reply:
The `m_amcIgnoreTTSList` is the `std::string` setting that lists which `TTS` inputs should be ignored on the backplane, and the `m_amcIgnoreTTS` is a `boolean` to check if this needs to happen.
However, I see now that it's not done correctly in the setting step, so I will change `m_ignoreAMCTTS` from `boolean` to `uint32_t` and write that to the `ttsDisableMask` function